node_modules/oae-core/about/about.html

```diff
@@ -0,0 +1,48 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/about.css" />
+
+<div class="about-widget">
+    <div id="about-header"><!-- --></div>
+    <div id="about-description"><!-- --></div>
+    <div id="about-members"><!-- --></div>
+</div>
+
+<div id="about-header-template"><!--
+    ${header('__MSG__ABOUT__', action)}
+--></div>
+
+<div id="about-description-template"><!--
+    {if contextProfile.description}
+        <div class="about-description-content">
+            ${oae.api.util.security().encodeForHTMLWithLinks(contextProfile.description).replace(/\n/g, '<br/>')}
```

Comment:
I've recently discovered that you can use `white-space: pre-wrap;` or `white-space: pre-line;` to ensure that line breaks are preserved, removing the need to do the replace. I'd be in favor of using that here and whilst we're at it also introduce it in the few other places where we're doing this

http://stackoverflow.com/questions/19684708/preserve-line-breaks-in-angularjs